### PR TITLE
fix(ihe): get metriport content from doc content

### PR DESCRIPTION
**Pull request by Orta21**
Ref. metriport/metriport-internal#1350

### Dependencies

- Upstream: none
- Downstream: npne

### Description

[Context](https://metriport.slack.com/archives/C04GEQ1GH9D/p1709670094144059) - we werent sending doc refs in the webhooks because we were checking if the doc didnt have CW content which was always true 

### Testing

_[Regular PRs:]_

- Local
  - [x] Was able to receive docs in the webhook
- Staging
  - [ ] Check if we receive docs in webhook
- Production
  - [ ] Check if we receive docs in webhook
  

### Release Plan

- [ ] Merge this

### CODE REVIEW

File: packages/utils/src/open-search/re-populate.ts [hunk 1]
```diff
@@ -120,32 +120,9 @@ function isFromCommonWell(doc: DocumentReference) {
...
-  // So, return true if it's explicitly Metriport or is not explicitly CommonWell.
-  return content.extension?.some(isMetriportExtension) === true || !isCommonwellContent(content);
-}
-function isCommonwellContent(content: DocumentReferenceContent): boolean {
-  return (
-    content.extension?.some(isCommonwellExtension) === true ||
-    content.attachment?.url?.includes("commonwellalliance.org") || // Legacy FHIR resources only have this
-    false
-  );
-}
```
Comment (Goncharo) [inline packages/utils/src/open-search/re-populate.ts:148]: is all this stuff unused now? I don't see where `isCommonwellExtension` or `isCommonwellContent` is defined

File: packages/core/src/external/fhir/shared/extensions/metriport.ts [hunk 2]
```diff
@@ -26,5 +28,23 @@ export function isMetriportContent(content: DocumentReferenceContent): boolean {
   // All doc refs created before this extension was added will have only one content element,
   // stored on S3 (Metriport) and w/o the extension.
   // So, return true if it's explicitly Metriport or is not explicitly CommonWell.
```
Comment (Goncharo) [inline packages/core/src/external/fhir/shared/extensions/metriport.ts:30]: this needs to be updated

File: packages/core/src/external/fhir/shared/extensions/metriport.ts [hunk 3]
```diff
@@ -26,5 +28,23 @@ export function isMetriportContent(content: DocumentReferenceContent): boolean {
...
-  return content.extension?.some(isMetriportExtension) === true || !isCommonwellContent(content);
+  return (
+    content.extension?.some(isMetriportExtension) === true ||
+    (!isCommonwellContent(content) && !isCarequalityContent(content))
+  );
```
Comment (Goncharo) [inline packages/core/src/external/fhir/shared/extensions/metriport.ts:34]: don't need the `=== true` here

File: packages/utils/src/open-search/re-populate.ts (hunk 1 above)
Comment (Orta21) [reply packages/utils/src/open-search/re-populate.ts:148]: Yea its unused since now its being imported (and used in the imported file)

**Review by Goncharo — APPROVED** [review]
:shipit:

**Pull request merged**